README.md

```diff
@@ -106,6 +106,34 @@ buildctl build ... --exporter=local --exporter-opt output=path/to/output-dir
 buildctl du -v
 ```
 
+#### Running containerized buildkit
+
+buildkit daemon supports communicating with clients over TCP which makes it possible, among other things, to run buildkit within _Docker for Mac_. 
+
+To run daemon in a container:
+
+```
+$ docker run --rm --privileged -p 12345:12345 --tmpfs /tmp  buildkit:buildd-standalone buildd-standalone --addr tcp://0.0.0.0:12345 --root /tmp/buildkit
+```
+
+Notes:
+
+* `--privileged` is needed for buildkit to access docker engine
+* `--tmpfs` is mandatory to provide support for [overlayfs](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/)
```

Comment:
`You need to make sure the --root directory exists on a filesystem that is supported by overlay filesystem. e.g. tmpfs, ext4, xfs.`